cmd/nvidia_gpu/nvidia_gpu.go

```diff
@@ -31,15 +31,20 @@ const (
 )
 
 var (
-	hostPathPrefix      = flag.String("host-path", "/home/kubernetes/bin/nvidia", "Path on the host that contains nvidia libraries. This will be mounted inside the container as '-container-path'")
-	containerPathPrefix = flag.String("container-path", "/usr/local/nvidia", "Path on the container that mounts '-host-path'")
-	pluginMountPath     = flag.String("plugin-directory", "/device-plugin", "The directory path to create plugin socket")
+	hostPathPrefix               = flag.String("host-path", "/home/kubernetes/bin/nvidia", "Path on the host that contains nvidia libraries. This will be mounted inside the container as '-container-path'")
+	containerPathPrefix          = flag.String("container-path", "/usr/local/nvidia", "Path on the container that mounts '-host-path'")
+	hostVulkanICDPathPrefix      = flag.String("host-vulkan-icd-path", "/home/kubernetes/bin/nvidia", "Path on the host that contains the Nvidia Vulkan installable client driver. This will be mounted inside the container as '-container-vulkan-icd-path'")
+	containerVulkanICDPathPrefix = flag.String("container-vulkan-icd-path", "/etc/vulkan/icd.d", "Path on the container that mounts '-host-vulkan-icd-path'")
```

Comment:
Do you know if vulkan/icd.d is also installed in order drivers? If the mount path doesn't exist, would a cuda container fail to start?

Reply:
I checked using the oldest supported supported driver, 390.46, and that creates vulkan/icd.d/nvidia_icd.json.

Also, I tested injecting a mount path that doesn't exist on the host and instead of gpu containers failing to start, the path was made on the host. The pod also wasn't privileged.

Reply:
I tested using the non-existing path behavior using this device plugin image:  gcr.io/chardch-gke-dev/nvidia-gpu-device-plugin:1.0